[core]/es_extended/server/main.lua

```diff
@@ -393,7 +393,7 @@ AddEventHandler("esx:playerLoaded", function(playerId, xPlayer, isNew)
 end)
 
 AddEventHandler("esx:setJob", function(src, job, lastJob)
-    ESX.JobsPlayerCount[lastJob.name] = (ESX.JobsPlayerCount[lastJob.name] or 1) - 1
+    ESX.JobsPlayerCount[lastJob.name] = ((ESX.JobsPlayerCount[lastJob.name] and ESX.JobsPlayerCount[lastJob.name] > 0) and ESX.JobsPlayerCount[lastJob.name] or 1) -1
```

Comment:
It's okey, but please have a variable for it because you use it more than once.

For example
local currentJobCount = ESX.JobsPlayerCount[job]
ESX.JobsPlayerCount[job] = ((currentJobCount and currentJobCount > 0) and currentJobCount or 1) -1